compiler/tests-io/dune

```
@@ -1,84 +1,72 @@
-(executables
- (names cat md5)
- (modes
-  js
-  wasm
-  (best exe)))
-
-(rule
- (target cat-native.stdout)
+(tests
+ (names cat)
+ (modes js wasm)
  (action
-  (with-stdout-to
-   %{target}
-   (pipe-stdout
-    (run printf "echo ☠")
-    (run %{dep:./cat.exe})))))
+  (pipe-stdout
+   (run printf "echo ☠")
+   (run node %{test}))))
```

It would be nice if dune could be updated so that we can just write:
```suggestion
   (run %{test}))))
```
But this does not seem straightforward...